src/view.ts

```diff
@@ -115,29 +115,44 @@ export const divider = (): DividerBlock => {
     }
 };
 
-export const homeView = (avg: string, completedCount: number, items: string[]) => {
+const formLink = "https://docs.google.com/forms/d/e/1FAIpQLSe1SltHxH47haVZzKe1x6eLsC89WmdEWtOTr_jo1sxg9t-jQw/viewform?usp=sf_link";
+
+export const homeView = (avg: string, completedCount: number, items: any[]) => {
```

Comment:
Is there a stronger type we could apply to `items: any[]`?

Reply:
whoops. Created it and didn't apply it here. Will do